analysis.md

```diff
@@ -0,0 +1,77 @@
+# Malware Analysis
+
```

Comment:
This needs something of a summary. I'll give you a template and you can customize.

> I'm trying to learn more about PowerShell so I can apply the knowledge towards feature engineering for a machine learning pipeline. I asked for interesting samples and my manager sent me a malicious Word document which had some PowerShell. The file hash of the document is `...`.
> 
> In the article below, I'll detail the results of my analysis including the methods and thought processes I used to get to each step.